### PR TITLE
Satellite: log the project information for upload, download, and list activities

### PR DESCRIPTION
**Pull request by Fadila82**
What: Log the Project ID in the logs for all upload, download, and list operations the satellite authorizes. 

Why: to identify which users utilize the network the most.

see [V3-3144](https://storjlabs.atlassian.net/browse/V3-3144)

Please describe the tests:
 - Test 1: start the storj-sim network using `storj-sim run | grep "satellite\0"` to only logs on the satellite. Upload, download or list objects should make the action appear in the satellite log at info level.

 - Test 2:
 
Please describe the performance impact: None

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: docs/blueprints/linux-setup.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+# Storage node installation and update on Linux
```
Comment (kaloyan-raev) [inline docs/blueprints/linux-setup.md]: This file should not be part of this PR.

File: satellite/metainfo/metainfo.go [hunk 2]
```diff
@@ -1262,6 +1262,7 @@ func (endpoint *Endpoint) GetObject(ctx context.Context, req *pb.ObjectGetReques
 			index++
 		}
 	}
+	endpoint.log.Sugar().Info("Authorize get object for project ID ", keyInfo.ProjectID)
```
Comment (kaloyan-raev) [inline satellite/metainfo/metainfo.go]: Valid for all changes:

1. Can we use structured logging, i.e.
```go
endpoint.log.Info("Get Object", zap.Stringer("Project ID", keyInfo.ProjectID))
```
2. Can we have the log message right after `keyInfo` variable is retrieved instead of at the end of the method?

3. Can we have the log message for all endpoint methods?

Comment (Fadila82) [reply satellite/metainfo/metainfo.go]: 1. Sure
2. The idea was to have the log only if there has been no error. If there is, it is already on the log. But I can add it :)
3. Sure. But is it not going to be too much for an Info log? I don't know how it will look like on a real satellite, but I'm afraid it won't be readable.

Comment (kaloyan-raev) [reply satellite/metainfo/metainfo.go]: OK, you are right for 2 and 3.
